README.md

```diff
@@ -58,30 +58,30 @@ The results here-below were computed on January the 30th, 2024 with the followin
 
 | Library      | Version  |
 |--------------|----------|
-| avaje-jsonb  | 1.9      |
+| avaje-jsonb  | 1.11     |
```

Comment:
These are lib versions for the published results. They must be updated when test result charts below are updated. Please revert this file.

Reply:
Done → 47fe058 (i hope it is OK to have this as a seperate commit, otherwise kindly advise and i can amend the individual commits)